ckcp/openshift_dev_setup.sh

```diff
@@ -14,10 +14,10 @@ WORK_DIR="$SCRIPT_DIR/work"
 KUBECONFIG=${KUBECONFIG:-$HOME/.kube/config}
 KUBECONFIG_KCP="$WORK_DIR/credentials/kubeconfig/kcp/admin.kubeconfig.base"
 CR_TO_SYNC=(
-            conditions.tekton.dev
+            services
```

Comment:
Since deployments are present in register.sh, we should include them here too to maintain consistency.